backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -402,7 +402,7 @@ func initDBDriver(driverName string, initConnectionTimeout time.Duration) string
 		dbName = common.GetStringConfig(mysqlDBName)
 	case "pgx":
 		sqlConfig = client.CreatePostgreSQLConfig(
-			common.GetStringConfigWithDefault(postgresUser, ""),
+			common.GetStringConfigWithDefault(postgresUser, "user"),
 			common.GetStringConfigWithDefault(postgresPassword, ""),
```

Comment:
Set password default value as `password` here? reference: https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/third-party/postgresql/base/pg-secret.yaml#L7

Reply:
Good point! Added the password.